operator/konflux.bundle.Dockerfile

```diff
@@ -63,6 +63,7 @@ RUN echo "Checking required RELATED_IMAGE_CENTRAL_DB"; [[ "${RELATED_IMAGE_CENTR
 RUN mkdir -p build/ && \
     rm -rf build/bundle && \
     cp -a bundle build/ && \
+    cp -v ../config-controller/config/crd/bases/config.stackrox.io_securitypolicies.yaml build/bundle/manifests/ && \
```

Comment:
ad Topic 1: The PR https://github.com/stackrox/stackrox/pull/13264 was set up independently but when I noticed that the PR at hand also requires this change to produce installable bundles, I had to cherry-pick this commit into this PR as well, to make it testable. Of course, when https://github.com/stackrox/stackrox/pull/13264 is merged, I can rebase this PR so that the commit in question is removed. Feel free to give me an approval for https://github.com/stackrox/stackrox/pull/13264.

Reply:
ad Topic 2: I have added the `backport-for-4.6-konflux-release` label, also to https://github.com/stackrox/stackrox/pull/13264. I suggest to merge https://github.com/stackrox/stackrox/pull/13264 first and then I will rebase this PR.

I am now aware of any recent changes to the CRD. @kylape and potentially others should know about such changes. I think the general flows for cherry-picking such changes to the released candidates should apply, this should in my opinion not be in-scope for these Konflux-specific changes.

Reply:
(Discussed in a meeting.)
I approved <https://github.com/stackrox/stackrox/pull/13264>.